app/views/alchemy/admin/pictures/_picture_to_assign.html.erb

```diff
@@ -18,7 +12,7 @@
     action_url,
     remote: true,
     onclick: '$(self).attr("href", "#").off("click"); return false',
-    method: action_method,
+    method: @action_method,
```

Comment:
Can we solve this without the awkwardly named `@action_method` ivar? It seems like it's only used in this one sport really, so I'd suggest: 
```
method: @content.blank? ? 'post' : 'put',
```